docs/CONNECTOR_PROTOCOL.md

```diff
@@ -191,17 +140,80 @@ In addition to the .elastic-connectors index for documents, we have an index to
     "worker_hostname" : { "type" : "keyword" },
     "indexed_document_count" : { "type" : "integer" },
     "deleted_document_count" : { "type" : "integer" },
-    "index_name": { "type": "keyword" },
```

Comment:
Why is index_name removed here? This was added because we don't want to cross-reference the connectors index to get this data, because that can be out of date. That's the same reason the connector document should be added (even though we never actually got around to implementing that).

If connectors aren't currently setting this data, they should.

Reply:
The index_name we should definitely keep, it's very low-effort and provides a good way to filter on index_name jobs rather than connector_id logs. If connectors aren't adding it now, we should fix that (or file an issue to fix it) instead of changing the protocol, I think. 

As for the connector object, you're right that that may require some extra thought and discussion. Given that we're not encryption configuration objects for now, the added risk is significant, I hadn't thought of that. Probably we can remove it for now and revisit it when we add queuing support.

Reply:
Agree, but for a technical preview, we don't need to consider backward-compatibility, and we even don't need to migrate to `v2`.